tests/unit_tests/test_wave_06.py

```diff
@@ -110,7 +110,18 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
```

Comment:
Good job with these! You could also add assertions to check that certain items are not in certain vendors' inventories:
```python
    assert item_c not in tai.inventory
    assert item_f not in jesse.inventory
```